### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by Laylo309**
- [ ] Add styles
- [ ] Change CSS files
- [ ] No linter errors
- [ ] Try to match design

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Laylo309, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 1]
```diff
@@ -1,17 +1,19 @@
-@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap");
 
 body {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
-  font-family: 'Montserrat', sans-serif;
+  font-family: "Montserrat", sans-serif;
```
Comment (mke2111) [inline src/index.css:7]: - [ ] Maintain the same font for your page, The font of Update progress button should be the one you maintain.

File: src/index.css [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
   overflow-x: hidden;
   background-color: rgba(0, 0, 0, 0.05);
 }
```
Comment (mke2111) [inline src/index.css:10]: - [ ] The entire body should have a bg color with the book container centered in between.

File: src/components/BooksContainer.js [hunk 3]
```diff
@@ -2,38 +2,45 @@
...
-          <Card.Text>Completed</Card.Text>
+          <Card.Title style={{ fontSize: '3rem', fontFamily: 'Montserrat' }}>65%</Card.Title>
+          <Card.Subtitle className="mb-2 text-muted">Completed</Card.Subtitle>
         </Card.Body>
-        <Card.Body>
-          <Card.Subtitle className="mb-2 text-muted">Current chapter</Card.Subtitle>
-          <Card.Title>Chapter 17</Card.Title>
+        <Card.Body className="chapter" style={{ width: '30%' }}>
+          <Card.Subtitle className="mb-2 text-muted" style={{ fontFamily: 'RobotoSlab' }}>CURRENT CHAPTER</Card.Subtitle>
+          <Card.Title style={{ fontFamily: 'RobotoSlab' }}>Chapter 17</Card.Title>
```
Comment (mke2111) [inline src/components/BooksContainer.js:43]: - [ ] The progress bar should have the completed section in color blue and not white. Follow the given design.

File: src/index.css (hunk 1 above)
Comment (Laylo309) [reply src/index.css:7]: Done ✅

File: src/components/BooksContainer.js (hunk 3 above)
Comment (Laylo309) [reply src/components/BooksContainer.js:43]: Done ✅

**Review by mke2111 — APPROVED** [review]
### Hi @Laylo309, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**